apollo-spaceport/proto/reports.proto

```diff
@@ -0,0 +1,388 @@
+syntax = "proto3";
```

Comment:
Can you include documentation somewhere (in the README?) of how to update this file? (The answer being that you can download it from https://usage-reporting.api.apollographql.com/proto/reports.proto)

Reply:
Yes. I can also investigate pulling and post-processing (to remove the [(js_use_toArray)=true] text from the file) as part of the build. We already have a build task that processes the local file, so requiring it to pull a remote file is not such a big deal.

Reply:
ah right, forgot about removing the js_use_toArray part